### ISSUE TITLE
me/repo should should show private repos

### ISSUE THREAD

**Issue by blaenk** [ProLoser/Github-Omnibox#59]
So I know about `gh /repo`, but sometimes I forget and I type in `gh blaenk/repo`, but it doesn't suggest private repos, even though I am blaenk. Given that the extension has authorization, isn't there a way for it to know that I'm blaenk, so that even if I use the `blaenk/repo` syntax it can also list the private repos?

**Comment by ProLoser** [comment]
Yeah, makes sense. I sorta want to go back to exploring atomic searching, where you don't have to do stuff like `name/whatever` or `/whatever`, you just type in `whatever` and it will search across all caches (owned repos, starred repos, users) and simply prioritize them by what would likely be most relevant. 

For this specific behavior, i was thinking about making it recognize that `blaenk/` is your name and resetting back to `/` for you but this would show all repos you have access to that might be under an org and I think it might be better to show just repos that are under your namespace. So yeah we should see if we can get that request that gets executed to include private repos.
